filtering-tags-problem/main.py

```diff
@@ -0,0 +1,81 @@
+import argparse
+import os
+from core import azure_wrapper, scheduler
+
+
+def parse_operation():
+    if cli_options["list_blobs"]:
+        return "list_blobs"
+    if cli_options["delete_blobs"]:
+        return "delete_blobs"
+    elif cli_options["download_blobs"]:
+        return "download_blobs"
+    elif cli_options["add_tag"]:
+        return "add_tag"
+    elif cli_options["update_tag"]:
+        return "update_tag"
+    elif cli_options["delete_tag"]:
+        return "delete_tag"
+    else:
+        raise Exception('Invalid option chosen')
+
+
+def main():
+    credentials = {}
+    if cli_options["connection_string"]:
+        credentials["connection_string"] = os.getenv("AZURE_STORAGE_CONNECTION_STRING")
```

Comment:
It seems we're reading the env variable and the passed argument is being ignored?

Reply:
We were thinking of supporting multiple authentication methods. 
So `cli_options["connection_string"]` is supposed to be a boolean variable. If user specifies login via connection_string, we read it from the environment variables.